.ci/check_python_dists.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Please revert this. This script is run on Windows and we've intentionally kept it using only `sh` to not have to worry about installing bash for Windows.

Just remove the `-E` instead.